force-app/main/default/classes/ReviewHandlerTest.cls

```diff
@@ -0,0 +1,98 @@
+@isTest
+public class ReviewHandlerTest {
+
+    @isTest
+    static void testPreventReviews() {
+        // Create a sample application with 'Rejected' status
+        Application__c rejectedApp = new Application__c(Status__c='Rejected');
+        insert rejectedApp;
+
+        // Create a review for the rejected application
+        Review__c review = new Review__c(Application__c=rejectedApp.Id, Status__c='Approved');
```

Comment:
use common method to create record as u need to create record in multiples places

Reply:
ok sir.